### ISSUE TITLE
Selecting many messages and starring or deleting one of them sometimes causes the message list to jump around

### ISSUE THREAD

**Issue by cobrabr** [leggett/simplify#832]
Selecting many (15+) messages and then trying to delete or star one of them using the respective icons causes the message list to scroll wildly. It seems to only happen when there are both many messages selected _and_ many messages overall on the list.

I'll try to capture a video of it happening if I can, but it's not consistently reproducible, unfortunately.

**Comment by leggett** [comment]
Thanks for the report... can you add a system report? 

You can grab it here:
https://mail.google.com/mail/?simplify=debug&do=copySystemReport

**Comment by cobrabr** [comment]
Sure thing, here you go:

```
Simplify v.3.1.23 u.0 configuration: aAX appNavOff avatarsInList avatarsZoom bundleInbox caA caI caL cfC cfF cfO cfR cfU composeActions composeFab darkTheme dateGroup fontList fontMsg fontNav gaArchive gaRead gaRemove geminiMin genSumMin groupHideFirst groupInbox groupOtherLists hideAds hideEmptySections hideExtWarn hideInboxHideAllLink hideLabelChips hideListCount hideMsgCount hideMsgReplies hideTabIcons idc8_343 importantBadge inInbox inList invertCompose isFirefox izBg7 listWidthLg lowDensity maTask matYou msgWidthSm nPane navOpen newNav newUI rightImportance rightLabels rightStar sendLater simplify simplifyPrefsOpen wjgnfnghs- 

System: Mozilla/5.0 (Windows NT 10.0 Win64 x64 rv:132.0) Gecko/20100101 Firefox/132.0 - gmail.pinto-server_20241024.02_p0 - Window: 1670 x 976 (116pct zoom) - Language: en-GB
```

**Comment by cobrabr** [comment]
@leggett This issue is still happening (and it's quite annoying). I have a video of the bug occurring, if that helps with anything, but I'd rather send it directly to you rather than have it publicly available here on YouTube, which is why I didn't just attach it here.

Please let me know if you'd like me to send you the video (and what your preferred way of getting it would be).

**Comment by leggett** [comment]
Yeah, Gmail made some changes that broke some fundamental things about managing focus.

This bug and a few others around the same time launched me on a massive re-write of bundles that's been going on ever since. 

It is mostly done but still not 100%. I launched what I have to the beta channel a couple weeks ago which you can try here: https://beta.simpl.fyi

**Comment by cobrabr** [comment]
Oh, sweet, I'll give that a try. Thanks!

EDIT: do you have a beta for Firefox?

**Comment by cobrabr** [comment]
@leggett Don't know if you saw my edit, but do you have a beta for Firefox?

**Comment by leggett** [comment]
Sorry I didn't reply. I don't have a beta for Firefox. But hopefully I'll launch the update to all users soon.
